projects/components/src/table/controls/table-controls.component.ts

```diff
@@ -257,10 +257,10 @@ export class TableControlsComponent implements OnChanges {
     );
   }
 
-  public onSelectChange(selectControl: TableSelectControl, selection: TableSelectControlOption): void {
+  public onSelectChange(selectControl: TableSelectControl, selection?: TableSelectControlOption): void {
```

Comment:
if this can be undefined, can you fix the typing on the select component also to show it can emit an undefined.

Reply:
done